doc/src/manual/profile.md

```diff
@@ -53,15 +53,16 @@ Now we're ready to profile this function:
 
 ```julia-repl
 julia> using Profile
+julia> using Profile: Time
 
-julia> @profile myfunc()
+julia> Time.@profile myfunc()
```

Comment:
I'm not sure we can do this change this so perhaps we should have a backwards compatible API. Something like

```
Profile.@memprofile
Profile.memprint
```

maybe.

Reply:
Yeah, this was just a quick fix to have all the functionality exposed. I'll have a look at a better api, maybe `@profile target=:memory ex` and have the other functions dispatch to the correct module based on the data that is being gathered.